browser/ui/tabs/brave_tab_color_mixer.cc

```diff
@@ -39,7 +41,12 @@ ChromeColorIds GetMappedChromeColorId(BraveColorIds brave_color_id) {
               {kColorBraveSplitViewTileBackground,
                kColorTabBackgroundInactiveFrameActive},
           });
-  return kChromiumColorMap.at(brave_color_id);
+
+  if (base::Contains(kChromiumColorMap, brave_color_id)) {
```

Comment:
I'm not the biggest fan of the pattern, but I think Chromium normally does a `.find` when they want to check whether an array contains something and return it to avoid the double lookup.

Up to you, though, I don't have strong feelings on it.

Reply:
Good call!